CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Upgrade to v0.7.1 of the Mailgun Terraform Provider
```

Comment:
Confirmed by looking at the upstream provider that there don't seem to be any new resources.